hri_safety_sense/src/VscProcess.cpp

```diff
@@ -103,6 +107,78 @@ VscProcess::~VscProcess()
 	if(joystickHandler) delete joystickHandler;
 }
 
+void VscProcess::receivedVibration(const std_msgs::Bool msg)
+{
+	bool received_msg = msg.data;
+	if (received_msg == true)
+	{
+		vsc_send_user_feedback(vscInterface, VSC_USER_BOTH_MOTOR_INTENSITY, MOTOR_CONTROL_INTENSITY_HIGH);
+	}
+}
+
+void VscProcess::receivedDisplayOnCommand(const hri_safety_sense::SrcDisplay& msg)
+{	
+	// Save the first message we receive as the previous message
+	static bool msg_received = false;
+	if (!msg_received)
+	{
+		prev_msg_ = msg;
+		msg_received = true;
+		vsc_send_user_feedback_string(vscInterface, VSC_USER_DISPLAY_ROW_1, msg.displayrow1.c_str());
+		vsc_send_user_feedback_string(vscInterface, VSC_USER_DISPLAY_ROW_2, msg.displayrow2.c_str());
+		vsc_send_user_feedback_string(vscInterface, VSC_USER_DISPLAY_ROW_3, msg.displayrow3.c_str());
+		vsc_send_user_feedback_string(vscInterface, VSC_USER_DISPLAY_ROW_4, msg.displayrow4.c_str());
+		return;
+	}
```

Comment:
The way this is coded, for the first message received, it will bypass all the MAXCHARACTERS checks, so no warnings will get printed.